release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -72,6 +72,27 @@ For more information, see xref:../installing/installing_aws/installing-aws-vpc.a
 [id="ocp-4-14-web-console"]
 === Web console
 
+[id="ocp-4-14-administrator-perspective"]
+==== Administrator Perspective
+
+With this release, there are several updates to the *Administrator* perspective of the web console. You can:
+
+* Narrow down the list of resources in a list view or search page with exact search capabilities. This will help when you have similarly named resources and fuzzy search doesn't narrow down your search.
+* Provide direct feedback about features and report a bug by clicking the *Help* button on the toolbar and click *Share Feedback* from the drop-down list.
+* Display and hide tooltips in the YAML editor. The tooltips will persist so you don't have to change it every time you navigate to the page.
+
+[id="supported-os-types-cluster"]
+===== Operating system based filtering in Operator Hub
+
+With this update, Operators in the Operator Hub are now filtered based on the nodes operating system since cluster can contain heterogenous nodes.
+
+[id="console-supports-installing-specific-operator-versions"]
+===== Support for installing specific operator versions in the web console
+
+With this update, you can then choose from a list of available versions for an Operator based on the selected channel on the *OperatorHub* page in the console. Additionally, you can view the metadata for that channel and version when available. When selecting an older version, a manual approval update strategy is required, otherwise the operator will immediately update back to the latest version on the channel.
```

Comment:
Nits:

```suggestion
With this update, you can now choose from a list of available versions for an Operator based on the selected channel on the *OperatorHub* page in the console. Additionally, you can view the metadata for that channel and version when available. When selecting an older version, a manual approval update strategy is required, otherwise the Operator will immediately update back to the latest version on the channel.
```

Alternatively, I think you can just remove "then" without replacing it with "now".